src/Datadog.Trace/Agent/Transports/ApiWebRequest.cs

```diff
@@ -14,7 +14,7 @@ public ApiWebRequest(HttpWebRequest request)
 
             // Default headers
             _request.Headers.Add(AgentHttpHeaderNames.Language, ".NET");
-            _request.Headers.Add(AgentHttpHeaderNames.TracerVersion, TracerConstants.AssemblyVersion);
+            _request.Headers.Add(AgentHttpHeaderNames.TracerVersion, TracerConstants.Version.ToString());
```

Comment:
This adds a string allocation at every request, could we cache this somewhere? (probably in TracerConstants)